pa11ycrawler/reporter.py

```diff
@@ -209,7 +209,11 @@ def make_html(self):
             result_file = os.path.join(self.results_dir, info['filename'] + '.1.0.json')
 
             with open(result_file, 'r') as report:
-                results = json.load(report)
+                try:
+                    results = json.load(report)
+                except ValueError as error:
+                    log.error(error.message)
```

Comment:
do you want to `error.message.decode('utf-8')` (or whatever the right implementation of that might be...)?